### DISCUSSION TITLE
Is there a UI for shop owner to see the orders coming in and manage them?

### DISCUSSION THREAD

**Discussion by hnurheryawan** [vuestorefront/shopware-pwa#1664] [Q&A] [answered]
### Description

Is there a UI for shop owner to see the orders coming in and manage them?

### Acceptance criteria

- [ ]

**Comment by develth** [comment, marked as answer]
Yes, it is called Shopware :)

https://docs.shopware.com/en/shopware-6-en/first-steps/administration-overview
